addons/account/models/account_move.py

```diff
@@ -790,6 +790,7 @@ def _compute_currency_id(self):
                 invoice.statement_line_id.foreign_currency_id
                 or invoice.journal_id.currency_id
                 or invoice.journal_id.company_id.currency_id
+                or invoice.company_currency_id
```

Comment:
I don't manage to reproduce this issue following the steps in the commit:
```
Create an invoice
Remove Journal
Traceback
```
I can't save because it's a required fields and thus, nothing is triggered and I don't get a traceback.

Am I missing something?

Reply:
Fixed in 326ae981b767a9ae3cacd50099dcce354df591d8